tensorflow/core/util/gpu_launch_config.h

```diff
@@ -357,6 +380,7 @@ inline const cudaStream_t& GetGpuStream(OpKernelContext* context) {
                                                 ->GpuStreamMemberHack()));
   return *ptr;
 }
+CREATE_CUDA_FUNCTION_ALIAS(GetGpuStream, GetCudaStream);
```

Comment:
We have changed `GetCudaStream` to `GetGpuStream` in our repo, but the definition is still within `#if GOOGLE_CUDA`, so we must not need / use it. It may sense to simply revert name back to `GetCudaStream`

Reply:
Agreed